Eggstravaganza/Assets/PlayerController.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerController : MonoBehaviour
+{
+    public float speed = 5f;
+    public Transform body;
+
+    public PlayerInputActions playerControls;
+
+    private InputAction move;
+    private InputAction interact;
+    private InputAction look;
+
+    public int[] eggInventory; //temporarily using an int, use Egg class later
+
+    private float stunTime = 1.5f;
+    bool isStunned = false;
+
+    private void Awake()
+    {
+        playerControls = new PlayerInputActions();
+    }
+    private void OnEnable()
+    {
+        move = playerControls.Player.Move;
+        move.Enable();
+
+        interact = playerControls.Player.Fire;
+        interact.Enable();
+        interact.performed += Interact;
+
+        look = playerControls.Player.Look;
+        look.Enable();
```

Comment:
I believe you can actually just enable the whole action map and then assign the individual actions as needed, like so:
```suggestion
        playerControls.Player.enable();
        move = playerControls.Player.Move;

        interact = playerControls.Player.Fire;
        interact.performed += Interact;

        look = playerControls.Player.Look;
```